core/modules/views/views.api.php

```diff
@@ -582,6 +582,8 @@ function hook_views_data_alter(&$data) {
  *     - uses grouping: Set to TRUE to allow the grouping settings for rows.
  *     - even empty: May have the value 'even empty' to tell Views that the style
  *       should be rendered even if there are no results.
+ *     - theme file: Name of the file containing theme or preprocess functions.
+ *     - theme path: Location of the theme file and template files needed.
```

Comment:
This description for theme path doesn't seem to be a proper sentence. Should that be "if needed" at the end of the sentence? It's also unclear if this path is relative to the Backdrop root or the module root.